src/script/media/MediaStreamHandler.js

```diff
@@ -275,16 +284,9 @@ export class MediaStreamHandler {
 
     return constraintsPromise
       .then(streamConstraints => this.requestMediaStream(mediaType, streamConstraints))
-      .then(mediaStreamInfo => {
-        // FIXME: the mediaStreamInUse should be more intelligent and handle all scenarios where the stream is actually needed
-        if (!isPreferenceChange && !this.mediaStreamInUse()) {
```

Comment:
This is a typical example of a place where the `mediaStreamHandler` needed to "guess" where the update came from (preferences or in the call). Now, because the `localMediaStream` of the `streamHandler` is **only** used for the current call, we will never need to care about preferences changes in here